build-docker.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -eu
+#!/usr/bin/env bash
+# Note: Avoid usage of arrays as MacOS users have an older version of bash (v3.x) which does not supports arrays
+set -eu
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
The use of `${BASH_SOURCE[0]}` isn't an issue?

Reply:
Yes you're right, good catch.

Reply:
Resolved by 6b630b400223fcaa24101e681b8716ba51df069e